### ISSUE TITLE
Field of view wrong but angle right

### ISSUE THREAD

**Issue by jfburke3** [mapillary/mapillary_issues#1742]
This image shows the field of view pointing in the wrong direction
http://www.mapillary.com/map/e/0puFK7SjZVao26Ao3tHq4w

However, if I edit the image, the angle-in-degrees looks right.  If I plus or minus it one bump, then suddenly the field of view is correct.  (I have left the image unchanged so you can play with it if you want.)

**Comment by Caboosey** [comment]
I am having same issues.

https://www.mapillary.com/map/im/2P_tNmm0mQUAP9wi2JbdmQ
Device used to capture photos: Samsung Galaxy S5 (Sprint) using Android 6.0

I even tried unchecking and rechecking the box to lock angle to forward direction within the Mapillary app.

**Comment by gyllen** [comment]
I believe this is the android issue for capturing.

**Comment by rsahlin** [comment]
Hi and thanks

Could you please let me know what version of the app you are using?

Best regards
/Richard

**Comment by Caboosey** [comment]
@rsahlin 

Maprillary version: 2.09
Android version: 6.0
Model: SM-G900P (Samsung Galaxy S5 Sprint)
Camera2 API: Checked
Lock direction forward in driving/biking mode: checked

I did noticed two errors in the log:
```
05-17 03:26:20.960 W/        (13006): Unable to open '/system/framework/com.qti.location.sdk.jar': No such file or directory
05-17 03:26:20.960 W/art     (13006): Failed to open zip archive '/system/framework/com.qti.location.sdk.jar': I/O Error
```
[Maprillary_logs.txt](https://github.com/mapillary/mapillary_issues/files/267759/Maprillary_logs.txt)

**Comment by rsahlin** [comment]
Fixed in 2.09

**Comment by ybkuang** [comment]
@jfburke3 This is a bug in the editor. We have two versions of compass angles. One from the device and one after the 3D reconstruction. In the map, we showed the one from device, but in the editing part we used the one from 3D reconstruction (87 degree) which is much better. We can click +/-, the angle on the map gets updated. That's why it looks like a jump. We're working on a new version of the editor. It will be resolved.
